cmd/podman/system/boot.go

```diff
@@ -0,0 +1,73 @@
+package system
+
+import (
+	"fmt"
+
+	"github.com/containers/common/pkg/completion"
+	"github.com/containers/podman/v2/cmd/podman/registry"
+	"github.com/containers/podman/v2/cmd/podman/utils"
+	"github.com/containers/podman/v2/cmd/podman/validate"
+	"github.com/containers/podman/v2/pkg/domain/entities"
+	"github.com/spf13/cobra"
+)
+
+var (
+	bootOptions     = entities.SystemPruneOptions{}
+	bootDescription = fmt.Sprintf(`
+  podman system boot
+
+        Start all stopped containers
+`)
+
+	bootCommand = &cobra.Command{
+		Use:               "boot",
+		Short:             "Boot stopped containers",
+		Args:              validate.NoArgs,
+		Long:              bootDescription,
+		RunE:              boot,
+		ValidArgsFunction: completion.AutocompleteNone,
+		Example:           `podman system boot`,
+	}
+)
+
+func init() {
+	registry.Commands = append(registry.Commands, registry.CliCommand{
+		Mode:    []entities.EngineMode{entities.ABIMode, entities.TunnelMode},
+		Command: bootCommand,
+		Parent:  systemCmd,
+	})
+}
+
+func boot(cmd *cobra.Command, args []string) error {
+	listOpts := entities.ContainerListOptions{
+		Filters: make(map[string][]string),
+		All:     true,
+	}
+	containers, err := registry.ContainerEngine().ContainerList(registry.GetContext(), listOpts)
+	if err != nil {
+		return err
+	}
+
+	cids := []string{}
+	for _, container := range containers {
+		if !container.Exited {
```

Comment:
Should not start any containers with a Restart Policy of NO.

Reply:
Sounds good but I am not sure where do I get this? Should I get this from the pod assigned to it?

Reply:
It might be as simple as
if container.ShouldRestart()